central/reports/scheduler/v2/reportgenerator/csv_gen.go

```diff
@@ -86,3 +82,29 @@ func GenerateCSV(cveResponses []*ImageCVEQueryResponse, configName string, inclu
 	}
 	return &zipBuf, nil
 }
+
+func addOptionalColumnstoHeader(optionalColumns *storage.VulnerabilityReportFilters) []string {
+	csvHeaderClone := make([]string, len(csvHeader))
+	copy(csvHeaderClone, csvHeader)
+	if optionalColumns.GetIncludeNvdCvss() {
+		csvHeaderClone = append(csvHeader, "NVDCVSS")
+	}
+	if optionalColumns.GetIncludeEpssProbability() {
+		csvHeaderClone = append(csvHeader, "EPSS Probability Percentage")
```

Comment:
QQ: Did we want to return this as a percentage in reports? Or did we want it to be the raw score?

Reply:
We show epss as % in UI. Hence we will show it as % in reports too